<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>osuSVToolClass</class>
 <widget class="QMainWindow" name="osuSVToolClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>541</width>
    <height>282</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>osuSVTool</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="NxSpeedChange">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculate Nx-looking speed change
(For bumps, glitches, etc...)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BPMNormalize">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>40</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Normalize BPM changes</string>
    </property>
   </widget>
   <widget class="QPushButton" name="MoveTimingPoints">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Move timing points</string>
    </property>
   </widget>
   <widget class="QPushButton" name="InheritUninherit">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>110</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Inherited &lt;-&gt; Uninherited
(Useful for osu! &lt;-&gt; BMS conversion)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="FlowingSpeedChanges">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Flowing speedups and slowdowns</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="osuSVTool.qrc"/>
 </resources>
 <connections/>
</ui>
